src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSyntaxFactsService.vb

```diff
@@ -206,6 +206,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return node.CheckParent(Of SimpleArgumentSyntax)(Function(p) p.IsNamed AndAlso p.NameColonEquals.Name Is node)
         End Function
 
+        Public Function GetNameOfParameter(node As SyntaxNode) As SyntaxToken Implements ISyntaxFactsService.GetNameOfParameter
+            Return If(TryCast(node, ParameterSyntax)?.Identifier?.Identifier, New SyntaxToken())
```

Comment:
>New SyntaxToken() [](start = 78, length = 17)

nit: consider `Nothing` for alternative value (symmetrical with the `default` in C# code and consistent with other methods in this file, such as `GetIdentifierOfGenericName`)

Reply:
VB doesn't like that.  It infers ```SyntaxToken?``` as the type :(

Reply:
Decided to just return a `SynaxToken?` since that better reflects that we might not find a parameter.